modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectViewLocalServiceTest.java

```diff
@@ -272,8 +272,8 @@ public void testUpdateObjectView() throws Exception {
 		ObjectView objectView = _addObjectView();
 
 		objectView = _objectViewLocalService.updateObjectView(
-			objectView.getObjectViewId(), objectView.isDefaultObjectView(),
-			objectView.getNameMap(),
+			objectView.getExternalReferenceCode(), objectView.getObjectViewId(),
```

Comment:
It would be nice to assert the externalReferenceCode used to update the ObjectView. Please see this [example](https://github.com/liferay/liferay-portal/blob/5363ba79d6f5607b99bc803ceb6c7b50ff779058/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java#L1465C23-L1466)